sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/ApplicationGatewayRequestRoutingRule.java

```diff
@@ -389,6 +394,27 @@ interface DefinitionStages {
              */
             WithAttach<ParentT> withUrlPathMap(String urlPathMapName);
         }
+
+        /**
+         * The stage of an application gateway request routing rule definition allowing to associate the rule with a
+         * priority.
+         * @param <ParentT> the stage of the application gateway definition to return to after attaching this definition
+         */
+        interface WithPriority<ParentT> {
+            /**
+             * ({@link ApplicationGatewaySkuName#STANDARD_V2} and {@link ApplicationGatewaySkuName#WAF_V2} Only)
```

Comment:
I am not sure how this show up in javadoc, but suggest to make it plain sentence (maybe after "Specifies ...") than ()